modules/indexer/issues/util.go

```diff
@@ -89,9 +89,9 @@ func getIssueIndexerData(ctx context.Context, issueID int64) (*internal.IndexerD
 		return nil, false, err
 	}
 
-	var projectID int64
-	if issue.Project != nil {
-		projectID = issue.Project.ID
+	var projectIDs []int64
+	for _, project := range issue.Projects {
+    projectIDs = append(projectIDs, project.ID)
 	}
```

Comment:
Maybe a function is better? Just a suggestion. Then we can use it in other places if necessary.